src/e2e-test/java/io/cdap/plugin/utils/enums/ErrorHandlingOptions.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.utils.enums;
+
+/**
+ * ErrorHandlingOptions enum.
+ */
+public enum ErrorHandlingOptions {
+  SKIP_ON_ERROR("Skip on error"),
+  STOP_ON_ERROR("Stop on error");
```

Comment:
are we testing SKIP_ON_ERROR vs STOP_ON_ERROR behavior? Also, I couldn't find STOP_ON_ERROR being used anywhere

Reply:
Added tests for the same.

Reply:
Are we testing error scenarios with SKIP_ON_ERROR vs STOP_ON_ERROR?
i.e. error records are skipped if SKIP_ON_ERROR is set, but pipeline run fails if there are errors and STOP_ON_ERROR is set.